docs/getting-started/benchmark.md

```diff
@@ -0,0 +1,211 @@
+Sets      5195097.56          ---          ---         0.26012         0.24700         0.49500         0.63100    400230.15
+```
+
+### Reads
+`memtier_benchmark -s $SERVER_PRIVATE_IP --distinct-client-seed --hide-histogram --ratio 0:1 -t 60 -c 20 -n 200000`
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Gets      6078632.89   6078632.89         0.00         0.27177         0.26300         0.49500         0.62300    438616.86
+```
+
+### Pipelined Reads
+`memtier_benchmark -s $SERVER_PRIVATE_IP --ratio 0:1 -t 60 -c 5  -n 200000  --distinct-client-seed --hide-histogram --pipeline=10`
+
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Gets      8975121.86   8975121.86         0.00         0.32325         0.31100         0.52700         0.83900    647619.14
+```
+
+
+## Comparison with Garnet
+Microsoft Research recently released Garnet ([https://github.com/microsoft/garnet]),
+a remote cache store. Due to interest within the Dragonfly community, we decided to compare
+Garnet's performance with Dragonfly's. This comparison focuses on performance results
+and does not delve into architectural differences or Redis compatibility implications.
+
+<i>Note: Unfortunately, Garnet does not have aarch64 build available,
+therefore we run both Garnet and Dragonfly on x86_64 server
+`c6in.12xlarge`. We run Garnet via docker with host networking enabled via
+`docker run --network=host ghcr.io/romange/garnet:latest --port=6379` command.
+The docker container was built using the Garnet docker build file for ubuntu, located in their
+repository.</i>
+
+### Garnet on `c6in.12xlarge`
+
+Similarly to previous tests we run `memtier_benchmark` on `c7gn.16xlarge` with `cluster`
+placement policy for both instances. For writes we used the following command:
+
+```
+memtier_benchmark -s $SERVER_PRIVATE_IP --distinct-client-seed --hide-histogram --ratio 1:0 -t 60 -c 20 -n 200000
+```
+
+Similarly, for reads we used
+```
+memtier_benchmark -s $SERVER_PRIVATE_IP --distinct-client-seed --hide-histogram --ratio 0:1 -t 60 -c 20 -n 200000
+```
+
+and for pipelined reads we used
+
+```
+memtier_benchmark -s $SERVER_PRIVATE_IP --ratio 0:1 -t 60 -c 5  -n 2000000  --distinct-client-seed --hide-histogram --pipeline=10
+```
+
+Note that we increased number of requests to `2000000` per client connection in the latter case.
```

Comment:
why though? and why not in the above tests?

Reply:
the pipelining QPS was so high that `200000` finishes too quickly